patches/components-component_updater-component_updater_service.h.patch

```diff
@@ -1,12 +1,20 @@
 diff --git a/components/component_updater/component_updater_service.h b/components/component_updater/component_updater_service.h
-index 9de20a64577ba39d467207ef28058a03bc7fb218..6131fd7f0810c4082b735b04d8add24baf4f4619 100644
+index 9de20a64577ba39d467207ef28058a03bc7fb218..53004cd57b593c5dc64081b8b446c397574aa3d7 100644
 --- a/components/component_updater/component_updater_service.h
 +++ b/components/component_updater/component_updater_service.h
-@@ -163,6 +163,7 @@ class OnDemandUpdater {
+@@ -150,6 +150,7 @@ class ComponentUpdateService {
+ 
+   friend class speech::SodaInstallerImpl;
+   friend class ::ComponentsHandler;
++  BRAVE_COMPONENT_UPDATER_SERVICE_H_
```

Comment:
@mkarolin could you please review my changes to patches, I renamed old macros to meet the class name because I need to create another one

Reply:
@spylogsster this one is fine, the other one I would probably go with BRAVE_COMPONENT_UPDATER_SERVICE_H_ON_DEMAND_UPDATER, or just BRAVE_ON_DEMAND_UPDATER, otherwise it looks like it's in a wrong file.

Reply:
done